gallery/how_to/work_with_relay/using_with_pipeline_executor.py

```diff
@@ -0,0 +1,182 @@
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Using Pipeline Executor in Relay
+=================================
+**Author**: `Hua Jiang <https://https://github.com/huajsj>`_
+
+This is a short tutorial on how to use the Pipeline Executor with Relay.
+"""
+import tvm
+from tvm import te
+import numpy as np
+from tvm.contrib import graph_executor as runtime
+from tvm import relay
+from tvm.relay import testing
+import tvm.testing
+import time
+
+#######################################################################
+# Create a simple network, this network can be a pre-trained model too.
+# ---------------------------------------------------------------------
+# Let's create a very simple network for demonstration.
+# It consists of convolution, batch normalization, and ReLU activation.
+def get_network():
+    out_channels = 16
+    batch_size = 1
+    data = relay.var("data", relay.TensorType((batch_size, 3, 224, 224), "float32"))
+    weight = relay.var("weight")
+    second_weight = relay.var("second_weight")
+    bn_gamma = relay.var("bn_gamma")
+    bn_beta = relay.var("bn_beta")
+    bn_mmean = relay.var("bn_mean")
+    bn_mvar = relay.var("bn_var")
+    simple_net = relay.nn.conv2d(
+        data=data, weight=weight, kernel_size=(3, 3), channels=out_channels, padding=(1, 1)
+    )
+    simple_net = relay.nn.batch_norm(simple_net, bn_gamma, bn_beta, bn_mmean, bn_mvar)[0]
+    simple_net = relay.nn.relu(simple_net)
+    simple_net = relay.nn.conv2d(
+        data=simple_net,
+        weight=second_weight,
+        kernel_size=(3, 3),
+        channels=out_channels,
+        padding=(1, 1),
+    )
+    simple_net = relay.Function(relay.analysis.free_vars(simple_net), simple_net)
+    data_shape = (batch_size, 3, 224, 224)
+    net, params = testing.create_workload(simple_net)
+    return net, params, data_shape
+
+
+net, params, data_shape = get_network()
+#############################################
+# Apply a customer graph splitting function.
+# ------------------------------------------
+# We use an testing linear graph splitting function as a example. User also can create their
+# own splitting function logic.
+import os
+
+os.sys.path.append(os.path.abspath(os.environ["TVM_HOME"] + "/tests/python/relay"))
```

Comment:
i think unfortunately right now this has to be done with relative paths. you can debug this with `tests/scripts/ci.py docs` i believe.

Reply:
@areusch , thanks for the follow up, the path issue get fixed, but seems like the ci box not enabled dnnl or not installed mkldnn?, then the tutorial still can not execute, to handle such issue, I put the BYOC part into a function and comment the function execution to avoid the DNNL execution error issue.

Reply:
it should have whatever is enabled in ci_gpu, and that's determined partly by Dockerfile.ci_gpu and by tests/scripts/task_config_build_gpu.sh. you could propose a change there if you need something for your tutorial (just add to this PR).

Reply:
@areusch, the Jekins file used a fixed docker image, the ci still running the tutorial file without apply the change in Dockerfile.ci_gpu.
I  can saw the new gpu docker  image get uploaded into aws ecr, and can not  found it on tlcstaging of docker hub, could I know what is process to request upload the new docker image to fix my issue?

Reply:
cc @driazati i think we need to set the ecr image repo to be public. thoughts?

Reply:
We could definitely do that and probably will soon. as a stop gap in the meantime @huajsj you can run the docker build locally and pass it to `ci.py`:
```bash
bash docker/build.sh ci_gpu --tag my_ci_gpu
python tests/scripts/ci.py docs --docker-image my_ci_gpu
```

Reply:
@driazati @areusch , thanks for the follow up, I tried the local test, and it work well, about this PR, to make it become green, do you think that is possible  to merge PR 11744 first ? which did the dnnl installation change and is the dependency this PR.